Documentation/doc/Documentation/advanced/Configuration_variables.txt

```diff
@@ -339,6 +341,15 @@ Note that the variables in the table below are being used.
 | `TBB_MALLOCPROXY_DEBUG_LIBRARY` | Full pathname of the compiled TBB debug malloc_proxy library (optional) | CMake |
 | `TBB_MALLOCPROXY_RELEASE_LIBRARY` | Full pathname of the compiled TBB release malloc_proxy library (optional) | CMake |
 
+\subsubsection installation_new_tbb OneTBB
+In the OneAPI version of \tbb, Intel provides cmake scripts of their own, and the user only have to set the `TBB_DIR`
+CMake variable at configuration time. A typical value for `TBB_DIR` is
+`/opt/intel/oneapi/tbb/latest/lib/cmake/tbb` on Unix systems, and `C:/dev/OneAPI/tbb/latest/lib/cmake/tbb` on Windows.
+The user will also have to source the `setvars` script that can be found at `/opt/intel/oneapi/setvars.sh` on Unix and
```

Comment:
> The user will also have to source the `setvars` script

No. If the user set `TBB_DIR` correctly, there is no need to source the `setvars` script. And if the user sources the script, then `TBB_DIR` is useless.